### ISSUE TITLE
[BUG] PC Search not working

### ISSUE THREAD

**Issue by Rempacious** [legendsayantan/MsRewards#24]
PC Search not working but mobile does. Tried the ff steps to try to fix but can't:
1. Re-login the account
2. Reset the app
3. Change the useragent

Tried both PC Search and Turbo Search

![Screenshot_2024-08-22-21-17-31-596_com legendsayantan msrewards](https://github.com/user-attachments/assets/089a4194-1991-46ae-826e-642240c3c4ce)

**Comment by legendsayantan** [comment]
Which engine are you using of Turbo?? Does changing it help?

**Comment by p2s246** [comment]
Same!! wanted to report that PC search is not working and it uses the daily quota in the app but not reflecting on the bing points.

**Comment by legendsayantan** [comment]
> Which engine are you using of Turbo?? Does changing it help?

I would require this info to help...

**Comment by p2s246** [comment]
> > Which engine are you using of Turbo?? Does changing it help?
> 
> I would require this info to help...

Okay, so I tried using the turbo search feature. Initially I was doing it from the main page by clicking the "play" button for PC searches and it didn't work, it only worked for mobile search with that method. But when I used turbo search in efficient mode it worked fine. It was able to do PC searches fine. I will try in performant mode tomorrow and will report if any issues.  Thanks.
